examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -36,13 +36,16 @@
     "We evaluate the results obtained with two approaches: using the ALS recommender algorithm vs. a baseline of random recommendations. \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        #inputCols=["year_made", "text_features", "genres_features"],
</pre><p>Is this comment needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>What is the 1043 in the start of the "features" column? Can we remove it?</p>

Reply:
> What is the 1043 in the start of the "features" column? Can we remove it?

1043 is the length of the "features" vector. I added explanation about it.

Reply:
> Line #14.        #inputCols=["year_made", "text_features", "genres_features"],
> Is this comment needed?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/microsoft/recommenders/pull/1505/discussion/)_

removed